docs/source/conf.py

```diff
@@ -77,4 +77,8 @@
     'peekingduck.pipeline.nodes.model.posenetv1',
     'peekingduck.pipeline.nodes.model.efficientdet_d04',
     'peekingduck.pipeline.nodes.model.hrnetv1',
+    'peekingduck.pipeline.nodes.input.utils',
+    'peekingduck.pipeline.nodes.dabble.utils',
+    'peekingduck.pipeline.nodes.draw.utils',
+    'peekingduck.pipeline.nodes.output.utils',
```

Comment:
Exclude these as well:
- peekingduck.pipeline.nodes.dabble.zoningv1
- peekingduck.utils
- peekingduck.weights_utils ?

Reply:
disagree with removing `weights_utils`. Would be good for open-source users to know how the downloading is effecting their local copy of the peekingduck